src/cmap/utils.py

```diff
@@ -0,0 +1,228 @@
+"""Utility functions and sample images for testing colormaps.
+
+The sineramp and circlesineramp functions in this module are based on the MATLAB
+functions of the same name, written by Peter Kovesi and available from his website
+under the MIT License:
+
+https://www.peterkovesi.com/matlabfns/
+Copyright (c) 1995-2020 Peter Kovesi
+pk@peterkovesi.com
+"""
+
+
+from typing import Iterable, Sized, cast
+
+import numpy as np
+from typing_extensions import TypedDict
+
+
+def sineramp(
+    shape: tuple[int, int] | int = (256, 512),
+    amp: float = 0.05,
+    wavelen: float = 8,
+    power: float = 2,
+) -> np.ndarray:
```

Comment:
this is fun! did you port this yourself? I hadn't heard of Kovesi before, thanks for that!